src/main/java/com/mongodb/hibernate/internal/translate/AbstractMqlTranslator.java

```diff
@@ -130,6 +132,9 @@ abstract class AbstractMqlTranslator<T extends JdbcOperation> implements SqlAstT
 
     AbstractMqlTranslator(SessionFactoryImplementor sessionFactory) {
         this.sessionFactory = sessionFactory;
+        assertNotNull(
+                assertNotNull(sessionFactory.getServiceRegistry().getService(StandardServiceRegistryScopedState.class))
+                        .getConfiguration());
```

Comment:
This way we can be sure that we will be able to access `MongoConfiguration` from our translators if we need that.

Reply:
yeah, this is a placeholder. When we do need it, we could update it accordingly. Currently it is fine to be as it is now.
`requireService()` usage might be a little better, but given this is a placeholder, it doesn't matter.

Reply:
Done in a8198a18c9479e86bc17825c9e1621daf6459f97.